src/main/resources/synthetic.xml

```diff
@@ -201,6 +201,8 @@
                   description="XML with the fields to set on the CI"/>
         <property name="addToEnvironment" category="input" kind="boolean" required="false"
                   description="Switch to decide if to add the CI to an environment"/>
+        <property name="addDictToEnvironment" category="input" kind="boolean" required="false"
```

Comment:
Wouldn't it make more sense to reuse the `addToEnvironment` and automatically detect if the created CI is a container or a dict?